packages/transit-vehicle-overlay/src/utils/recenter.js

```diff
@@ -52,13 +68,13 @@ export function recenterPanToOffset(
 }
 
 /** callback used to move the map to some coordinates (e.g., selected vehicle) */
-export function recenterPanTo(pause = 700) {
+export function recenterPanTo(doRecenter = true, pause = 700) {
```

Comment:
This might be a breaking change. (It does not seem to be a breaking change for otp-react-redux.)